# ROS Topics
pose_topic: /mavros/local_position/pose
twist_topic: /mavros/local_position/velocity_local

# FSM Parameters
fsm_frequency: 100
message_timeout: 0.25
motor_speedup_time: 5.0
takeoff_height: 1.0
takeoff_land_speed: 0.3
reject_takeoff_twist_threshold: 0.15
hover_max_velocity: 3.0
hover_max_yaw_rate: 3.0
check_safety_volumn: true
safety_volumn: [-20.0, 20.0, -20.0, 20.0, -10, 10.0]

# RC Transmitter Parameters
without_rc: true
throttle_channel: 4
yaw_channel: 3
pitch_channel: 1
roll_channel: 2
fsm_channel: 5
command_channel: 7
reboot_channel: 8
reverse_throttle: false
reverse_yaw: false
reverse_pitch: false
reverse_roll: false
switch_threshold: 0.75
joystick_deadzone: 0.10
check_centered_threshold: 0.3